docker-compose.yml

```diff
@@ -11,7 +11,7 @@ services:
   api:
     container_name: api
     build: api
-    env_file: .env
+    env_file: ${CHOSEN_ENV}
```

Comment:
Just a minor thing but I personally would prefer `TARGET_ENV` as opposed to `CHOSEN_ENV`. Not a strong opinion so feel free to ignore this.

Reply:
I thought `TARGET_ENV` can be misinterpreted as "the env _of the_ target". So I purposefully opted for the admittedly less-common wording `CHOSEN_ENV` hoping that it would be clearer that as a developer I am deploying the stack for this chosen environment.
How does this explanation resonate with you?

Reply:
That sounds fine! I was really just nitpicking on the past perfect tense of the verb. Let's leave it as is for now.